VS_Project/Pokerapp/Pokerapp/Tableview.cshtml

```diff
@@ -133,51 +137,209 @@
                     <input class="btn btn-outline-secondary" type="submit" name="blindnext" value="Raise blinds" />
                 </form>
             </div>
-            <div class="gridd">
-                <div class="white"></div>
-                <div class="white2"></div>
-                <div class="white3"></div>
-                <div class="white4"></div>
-                @if (aantal <= 7)
-                {
-                    foreach (var a in db.Query("SELECT TOP (@2) Name FROM Players WHERE Room_Code = @0 AND Tbl_nr = @1 ORDER BY Name DESC", roomCode, table, tweede))
-                    {
-                        <div class="chair2">
-                            <img src="~/stoel.png" alt="chair" style="width:75px;height:75px" />
-                            <b>@a.Name</b>
+
+        </div>
+    </div>
+</div>
+<div id="activebody">
+    <div>
+        @foreach (var a in db.Query("SELECT TOP (@0) Name, Rebuy FROM Players WHERE Room_Code = @1 AND Tbl_nr = @2 AND Active = 1 ORDER BY Name ASC", eerste, roomCode, table, eerste))
+        {
+
+
+            string player = (string)db.QueryValue("SELECT Name FROM Players WHERE Name = @0 AND Room_Code = @1 AND Tbl_nr = @2", a.Name, roomCode, table);
+            string gaanmetdiebanaan = player + roomCode;
+
+            <!-- Button trigger modal -->
+            <button id="pokerstool" value="@player" name="player" type="button" data-toggle="modal" data-target="#@gaanmetdiebanaan">@player</button>
+
+
+
+            int rebuyyy = (int)db.QueryValue("SELECT Rebuy FROM Players WHERE Name = @0 AND Room_Code = @1 AND Tbl_nr = @2", player, roomCode, table);
```

Comment:
Normale variabel namen. 1 y is genoeg en anders descriptive namen geven. (rebuy1, rebuy2 whatever)